UIKitDz/UIKitDz/LableViewController.swift

```diff
@@ -0,0 +1,315 @@
+//
+//  ViewController.swift
+//  UIKitDz
+//
+//  Created by Valeriy Fomin on 21/6/21.
+//
+
+import UIKit
+
+final class LableViewController: UIViewController {
```

Comment:
```suggestion
final class LabelViewController: UIViewController {
```
но вообще по хорошему наверно лучше какое то название вроде TextEditorViewController, потому что Label не совсем про него